tedx-website/src/components/landing.css

```diff
@@ -0,0 +1,119 @@
+
+  html {
+    /* set background color */
+    background-color: black;
+  }
+  body{
+    background-color: transparent !important;
+    max-width: 100%;
+    overflow-x: hidden !important;
+  }
+  .grid{
+    margin-left: 10vh;   
+    margin-right: 10vh;
+  }
+  
+  .title{
+    font-size: 5vw;
+    color: white;
```

Comment:
Could we use `var(--text-color)` instead to avoid hardcoding values and maintain consistency in text color across the website? This comment applies to everywhere else that you use `white` in this file.